test/e2e/select.test.js

```diff
@@ -9,71 +9,136 @@ describe('vl-select', async () => {
+        await assert.eventually.isTrue(select.hasValue('België'));
+        await assert.eventually.isTrue(select.hasValue('Duitsland'));
+        await assert.eventually.isTrue(select.hasValue('Frankrijk'));
     });
 
-    it('ik kan een value selecteren', async () => {
+    it('als gebruiker kan ik een value selecteren', async () => {
         const select = await vlSelectPage.getDefaultSelect();
         await select.selectByValue('Duitsland');
-        const selectedValue = await select.getSelectedValue();
-        assert.isTrue(selectedValue === 'Duitsland');
+        await assert.eventually.equal(select.getSelectedValue(), 'Duitsland');
     });
-    
-    it('ik kan een select definieren als error', async () => {
-        const select = await vlSelectPage.getErrorSelect();
-        await assert.eventually.isTrue(select.isError());
+
+    it('als gebruiker kan ik een optie selecteren via tekst', async () => {
+        const select = await vlSelectPage.getDefaultSelect();
+        await select.selectByText('Frankrijk');
+        await assert.eventually.equal(select.getSelectedValue(), 'Frankrijk');
+    });
+
+    it('als gebruiker kan ik een optie selecteren via value', async () => {
+        const select = await vlSelectPage.getDefaultSelect();
+        await select.selectByValue('Duitsland');
+        await assert.eventually.equal(select.getSelectedValue(), 'Duitsland');
     });
     
-    it('ik kan een select definieren als success', async () => {
-        const select = await vlSelectPage.getSuccessSelect();
-        await assert.eventually.isTrue(select.isSuccess());
+    it('als gebruiker kan ik een optie selecteren via index', async () => {
+        const select = await vlSelectPage.getDefaultSelect();
+        await select.selectByIndex(2);
+        await assert.eventually.equal(select.getSelectedValue(), 'Frankrijk');
     });
     
-    it('ik kan controleren of een select disabled is of niet', async () => {
-        const select = await vlSelectPage.getDisabledSelect();
-        await assert.eventually.isTrue(select.isDisabled());
+    it('als gebruiker zie ik wanneer een select een error bevat', async () => {
+        const select = await vlSelectPage.getDefaultSelect();
+        const errorSelect = await vlSelectPage.getErrorSelect();
+        await assert.eventually.isFalse(select.isError());
+        await assert.eventually.isTrue(errorSelect.isError());
     });
     
-    it('ik kan een optie selecteren via het zoekveld', async () => {
+    it('als gebruiker zie ik wanneer een select succesvol is', async () => {
+        const select = await vlSelectPage.getDefaultSelect();
+        const successSelect = await vlSelectPage.getSuccessSelect();
+        await assert.eventually.isFalse(select.isSuccess());
+        await assert.eventually.isTrue(successSelect.isSuccess());
+    });
+
+    // TODO test dressed test
+
+    // TODO test faalt dus probleem bij page object
```

Comment:
bedoeling dat er TODOs achterblijven na dit ticket?

Reply:
+1

Reply:
Nee, deze PR is mijn review op de vl-ui-select, dus iemand (ook ik) kan deze TODO's nog invullen.